synapse/lib/cell.py

```diff
@@ -579,6 +589,13 @@ def addHttpApi(self, path, ctor, info):
         self.onfini(auth.fini)
         return auth
 
+    def _initCellApiCmds(self):
+        self.addCellCmdType('hive')
```

Comment:
Since all cells have hive and boss, lets name one of the interfaces "cell" ( again, with version info ) and have those commands load via the "cell" interface.